praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/service/MeldezeitraumService.java

```diff
@@ -17,6 +20,8 @@
 public class MeldezeitraumService {
     private final MeldezeitraumMapper meldezeitraumMapper;
     private final MeldezeitraumRepository meldezeitraumRepository;
+    private final AusbildungsPraktikumsstellenRepository ausbildungsPraktikumsstellenRepository;
+    private final StudiumsPraktikumsstellenRepository studiumsPraktikumsstellenRepository;
```

Comment:
Please use the `PraktikumsstellenService`. It should be the owner of all access to the repositories `StudiumsPraktikumsstellenRepository` and `AusbildungsPraktikumsstellenRepository`.

I know that the services uses `MeldezeitraumService`, but i am sure you will find a good solution for that.